pkg/client/client.go

```diff
@@ -10,9 +10,9 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hewlettpackard/hpegl-provider-lib/pkg/client"
 
+	rest "github.com/hewlettpackard/hpegl-metal-client/v1/pkg/client"
```

Comment:
I know there was a blank line here before, however since these are al coming from the hewlettpackard organization, they should not be separated.